### ISSUE TITLE
2.6.3 Can't work on versions under 1.12

### ISSUE THREAD

**Issue by TheLimeGlass** [SkriptLang/Skript#4933]
### Skript/Server Version

```markdown
2.6.3
Paper 1.11
```

### Bug Description

Sweeping edge doesn't exist.

### Expected Behavior

To enable on startup.

### Steps to Reproduce

Install 2.6.3 on 1.11
Was all a clean install too.

### Errors or Screenshots

```
[11:55:00 ERROR]: Error occurred while enabling Skript v2.6.3 (Is it up to date?)
java.lang.AssertionError: missing name for Enchantment[22, SWEEPING]
        at ch.njol.skript.bukkitutil.EnchantmentUtils.getKey(EnchantmentUtils.java:83) ~[?:?]
        at ch.njol.skript.util.EnchantmentType.lambda$static$0(EnchantmentType.java:118) ~[?:?]
        at ch.njol.skript.localization.Language.addListener(Language.java:331) ~[?:?]
        at ch.njol.skript.localization.Language.addListener(Language.java:322) ~[?:?]
        at ch.njol.skript.util.EnchantmentType.<clinit>(EnchantmentType.java:114) ~[?:?]
        at ch.njol.skript.classes.data.BukkitClasses.<clinit>(BukkitClasses.java:1322) ~[?:?]
        at ch.njol.skript.Skript.onEnable(Skript.java:498) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:271) ~[patched_1.11.2.jar:git-Paper-1106]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:316) [patched_1.11.2.jar:git-Paper-1106]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:407) [patched_1.11.2.jar:git-Paper-1106]
        at org.bukkit.craftbukkit.v1_11_R1.CraftServer.enablePlugin(CraftServer.java:378) [patched_1.11.2.jar:git-Paper-1106]
        at org.bukkit.craftbukkit.v1_11_R1.CraftServer.enablePlugins(CraftServer.java:328) [patched_1.11.2.jar:git-Paper-1106]
        at net.minecraft.server.v1_11_R1.MinecraftServer.t(MinecraftServer.java:438) [patched_1.11.2.jar:git-Paper-1106]
        at net.minecraft.server.v1_11_R1.MinecraftServer.l(MinecraftServer.java:399) [patched_1.11.2.jar:git-Paper-1106]
        at net.minecraft.server.v1_11_R1.MinecraftServer.a(MinecraftServer.java:337) [patched_1.11.2.jar:git-Paper-1106]
        at net.minecraft.server.v1_11_R1.DedicatedServer.init(DedicatedServer.java:276) [patched_1.11.2.jar:git-Paper-1106]
        at net.minecraft.server.v1_11_R1.MinecraftServer.run(MinecraftServer.java:612) [patched_1.11.2.jar:git-Paper-1106]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_291]
[11:55:00 INFO]: [Skript] Disabling Skript v2.6.3
[11:55:00 ERROR]: Error occurred while disabling Skript v2.6.3 (Is it up to date?)
java.lang.ExceptionInInitializerError: null
        at ch.njol.skript.Skript.beforeDisable(Skript.java:1116) ~[?:?]
        at ch.njol.skript.Skript.onDisable(Skript.java:1128) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:273) ~[patched_1.11.2.jar:git-Paper-1106]
        at org.bukkit.plugin.java.JavaPluginLoader.disablePlugin(JavaPluginLoader.java:344) [patched_1.11.2.jar:git-Paper-1106]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:320) [patched_1.11.2.jar:git-Paper-1106]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:407) [patched_1.11.2.jar:git-Paper-1106]
        at org.bukkit.craftbukkit.v1_11_R1.CraftServer.enablePlugin(CraftServer.java:378) [patched_1.11.2.jar:git-Paper-1106]
        at org.bukkit.craftbukkit.v1_11_R1.CraftServer.enablePlugins(CraftServer.java:328) [patched_1.11.2.jar:git-Paper-1106]
        at net.minecraft.server.v1_11_R1.MinecraftServer.t(MinecraftServer.java:438) [patched_1.11.2.jar:git-Paper-1106]
        at net.minecraft.server.v1_11_R1.MinecraftServer.l(MinecraftServer.java:399) [patched_1.11.2.jar:git-Paper-1106]
        at net.minecraft.server.v1_11_R1.MinecraftServer.a(MinecraftServer.java:337) [patched_1.11.2.jar:git-Paper-1106]
        at net.minecraft.server.v1_11_R1.DedicatedServer.init(DedicatedServer.java:276) [patched_1.11.2.jar:git-Paper-1106]
        at net.minecraft.server.v1_11_R1.MinecraftServer.run(MinecraftServer.java:612) [patched_1.11.2.jar:git-Paper-1106]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_291]
Caused by: ch.njol.skript.SkriptAPIException: Registration can only be done during plugin initialization
        at ch.njol.skript.Skript.checkAcceptRegistrations(Skript.java:1223) ~[?:?]
        at ch.njol.skript.Skript.registerEvent(Skript.java:1429) ~[?:?]
        at ch.njol.skript.events.EvtSkript.<clinit>(EvtSkript.java:41) ~[?:?]
        ... 14 more
```

### Other

Disable error is probably nothing to worry about, and probably only happens because Skript had an error actually starting.

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by TheLimeGlass** [comment]
We might have to release a 2.6.4 to fix.

**Comment by TheLimeGlass** [comment]
Here is an issue when running 1.10
```
[11:50:17 ERROR]: #!#! Stack trace:
[11:50:17 ERROR]: #!#! java.lang.RuntimeException: pattern compiling exception, element class: ch.njol.skript.effects.EffStopSound
[11:50:17 ERROR]: #!#!     at ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:226)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:177)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.lang.Statement.parse(Statement.java:60)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.ScriptLoader.loadItems(ScriptLoader.java:1114)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.ScriptLoader.loadScript(ScriptLoader.java:753)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.ScriptLoader.lambda$loadScripts$4(ScriptLoader.java:526)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.ScriptLoader.lambda$makeFuture$2(ScriptLoader.java:418)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.ScriptLoader.makeFuture(ScriptLoader.java:434)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.ScriptLoader.loadScripts(ScriptLoader.java:525)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.Skript$1.lambda$run$0(Skript.java:626)
[11:50:17 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_10_R1.scheduler.CraftTask.run(CraftTask.java:58)
[11:50:17 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_10_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:355)
[11:50:17 ERROR]: #!#!     at net.minecraft.server.v1_10_R1.MinecraftServer.D(MinecraftServer.java:798)
[11:50:17 ERROR]: #!#!     at net.minecraft.server.v1_10_R1.DedicatedServer.D(DedicatedServer.java:404)
[11:50:17 ERROR]: #!#!     at net.minecraft.server.v1_10_R1.MinecraftServer.C(MinecraftServer.java:740)
[11:50:17 ERROR]: #!#!     at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:639)
[11:50:17 ERROR]: #!#!     at java.lang.Thread.run(Thread.java:748)
[11:50:17 ERROR]: #!#! Caused by: ch.njol.skript.patterns.MalformedPatternException: caught exception while compiling pattern [pattern: stop sound[s] %strings% [(in|from) %-soundcategory%] [(from playing to|for) %players%]]
[11:50:17 ERROR]: #!#!     at ch.njol.skript.patterns.PatternCompiler.compile(PatternCompiler.java:54)
[11:50:17 ERROR]: #!#!     at java.util.HashMap.computeIfAbsent(HashMap.java:1127)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.lang.SkriptParser.parse_i(SkriptParser.java:1254)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:224)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:177)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.lang.Statement.parse(Statement.java:60)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.ScriptLoader.loadItems(ScriptLoader.java:1114)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.ScriptLoader.loadScript(ScriptLoader.java:753)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.ScriptLoader.lambda$loadScripts$4(ScriptLoader.java:526)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.ScriptLoader.lambda$makeFuture$2(ScriptLoader.java:418)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.ScriptLoader.makeFuture(ScriptLoader.java:434)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.ScriptLoader.loadScripts(ScriptLoader.java:525)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.Skript$1.lambda$run$0(Skript.java:626)
[11:50:17 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_10_R1.scheduler.CraftTask.run(CraftTask.java:58)
[11:50:17 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_10_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:355)
[11:50:17 ERROR]: #!#!     at net.minecraft.server.v1_10_R1.MinecraftServer.D(MinecraftServer.java:798)
[11:50:17 ERROR]: #!#!     at net.minecraft.server.v1_10_R1.DedicatedServer.D(DedicatedServer.java:404)
[11:50:17 ERROR]: #!#!     at net.minecraft.server.v1_10_R1.MinecraftServer.C(MinecraftServer.java:740)
[11:50:17 ERROR]: #!#!     at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:639)
[11:50:17 ERROR]: #!#!     at java.lang.Thread.run(Thread.java:748)
[11:50:17 ERROR]: #!#! Caused by: ch.njol.skript.SkriptAPIException: No class info found for soundcategory
[11:50:17 ERROR]: #!#!     at ch.njol.skript.registrations.Classes.getClassInfo(Classes.java:264)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.patterns.TypePatternElement.fromString(TypePatternElement.java:92)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.patterns.PatternCompiler.compile(PatternCompiler.java:138)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.patterns.PatternCompiler.compile(PatternCompiler.java:77)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.patterns.PatternCompiler.compile(PatternCompiler.java:49)
[11:50:17 ERROR]: #!#!     at java.util.HashMap.computeIfAbsent(HashMap.java:1127)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.lang.SkriptParser.parse_i(SkriptParser.java:1254)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:224)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.lang.SkriptParser.parse(SkriptParser.java:177)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.lang.Statement.parse(Statement.java:60)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.ScriptLoader.loadItems(ScriptLoader.java:1114)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.ScriptLoader.loadScript(ScriptLoader.java:753)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.ScriptLoader.lambda$loadScripts$4(ScriptLoader.java:526)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.ScriptLoader.lambda$makeFuture$2(ScriptLoader.java:418)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.ScriptLoader.makeFuture(ScriptLoader.java:434)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.ScriptLoader.loadScripts(ScriptLoader.java:525)
[11:50:17 ERROR]: #!#!     at ch.njol.skript.Skript$1.lambda$run$0(Skript.java:626)
[11:50:17 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_10_R1.scheduler.CraftTask.run(CraftTask.java:58)
[11:50:17 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_10_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:355)
[11:50:17 ERROR]: #!#!     at net.minecraft.server.v1_10_R1.MinecraftServer.D(MinecraftServer.java:798)
[11:50:17 ERROR]: #!#!     at net.minecraft.server.v1_10_R1.DedicatedServer.D(DedicatedServer.java:404)
[11:50:17 ERROR]: #!#!     at net.minecraft.server.v1_10_R1.MinecraftServer.C(MinecraftServer.java:740)
[11:50:17 ERROR]: #!#!     at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:639)
[11:50:17 ERROR]: #!#!     at java.lang.Thread.run(Thread.java:748)
[11:50:17 ERROR]: #!#!
[11:50:17 ERROR]: #!#! Version Information:
[11:50:17 ERROR]: #!#!   Skript: 2.6.3 (custom version)
[11:50:17 ERROR]: #!#!     Flavor: selfbuilt-unknown
[11:50:17 ERROR]: #!#!     Date: unknown
[11:50:17 ERROR]: #!#!   Bukkit: 1.10.2-R0.1-SNAPSHOT
[11:50:17 ERROR]: #!#!   Minecraft: 1.10.2
[11:50:17 ERROR]: #!#!   Java: 1.8.0_291 (Java HotSpot(TM) 64-Bit Server VM 25.291-b10)
[11:50:17 ERROR]: #!#!   OS: Windows 10 amd64 10.0
[11:50:17 ERROR]: #!#!
[11:50:17 ERROR]: #!#! Server platform: Paper
[11:50:17 ERROR]: #!#!
[11:50:17 ERROR]: #!#! Current node: assert max health of {_pig} = 20 with "pig failed to set max health to 20: Max health of pig = %max health of {_pig}%" (..\..\..\..\..\src\test\skript\tests\misc\2584-HealthUtils - update.sk, line 6)
[11:50:17 ERROR]: #!#! Current item: null
[11:50:17 ERROR]: #!#!
[11:50:17 ERROR]: #!#! Thread: Server thread
[11:50:17 ERROR]: #!#!
[11:50:17 ERROR]: #!#! Language: english
[11:50:17 ERROR]: #!#! Link parse mode: DISABLED
[11:50:17 ERROR]: #!#!
[11:50:17 ERROR]: #!#! End of Error.
```

**Comment by TheLimeGlass** [comment]
I added my gradle test environment to the dev/2.6 branch so I can grab these errors with Java 8

**Comment by TheLimeGlass** [comment]
1.11 issue
```
[11:55:00 ERROR]: Error occurred while enabling Skript v2.6.3 (Is it up to date?)
java.lang.AssertionError: missing name for Enchantment[22, SWEEPING]
        at ch.njol.skript.bukkitutil.EnchantmentUtils.getKey(EnchantmentUtils.java:83) ~[?:?]
        at ch.njol.skript.util.EnchantmentType.lambda$static$0(EnchantmentType.java:118) ~[?:?]
        at ch.njol.skript.localization.Language.addListener(Language.java:331) ~[?:?]
        at ch.njol.skript.localization.Language.addListener(Language.java:322) ~[?:?]
        at ch.njol.skript.util.EnchantmentType.<clinit>(EnchantmentType.java:114) ~[?:?]
        at ch.njol.skript.classes.data.BukkitClasses.<clinit>(BukkitClasses.java:1322) ~[?:?]
        at ch.njol.skript.Skript.onEnable(Skript.java:498) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:271) ~[patched_1.11.2.jar:git-Paper-1106]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:316) [patched_1.11.2.jar:git-Paper-1106]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:407) [patched_1.11.2.jar:git-Paper-1106]
        at org.bukkit.craftbukkit.v1_11_R1.CraftServer.enablePlugin(CraftServer.java:378) [patched_1.11.2.jar:git-Paper-1106]
        at org.bukkit.craftbukkit.v1_11_R1.CraftServer.enablePlugins(CraftServer.java:328) [patched_1.11.2.jar:git-Paper-1106]
        at net.minecraft.server.v1_11_R1.MinecraftServer.t(MinecraftServer.java:438) [patched_1.11.2.jar:git-Paper-1106]
        at net.minecraft.server.v1_11_R1.MinecraftServer.l(MinecraftServer.java:399) [patched_1.11.2.jar:git-Paper-1106]
        at net.minecraft.server.v1_11_R1.MinecraftServer.a(MinecraftServer.java:337) [patched_1.11.2.jar:git-Paper-1106]
        at net.minecraft.server.v1_11_R1.DedicatedServer.init(DedicatedServer.java:276) [patched_1.11.2.jar:git-Paper-1106]
        at net.minecraft.server.v1_11_R1.MinecraftServer.run(MinecraftServer.java:612) [patched_1.11.2.jar:git-Paper-1106]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_291]
[11:55:00 INFO]: [Skript] Disabling Skript v2.6.3
[11:55:00 ERROR]: Error occurred while disabling Skript v2.6.3 (Is it up to date?)
java.lang.ExceptionInInitializerError: null
        at ch.njol.skript.Skript.beforeDisable(Skript.java:1116) ~[?:?]
        at ch.njol.skript.Skript.onDisable(Skript.java:1128) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:273) ~[patched_1.11.2.jar:git-Paper-1106]
        at org.bukkit.plugin.java.JavaPluginLoader.disablePlugin(JavaPluginLoader.java:344) [patched_1.11.2.jar:git-Paper-1106]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:320) [patched_1.11.2.jar:git-Paper-1106]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:407) [patched_1.11.2.jar:git-Paper-1106]
        at org.bukkit.craftbukkit.v1_11_R1.CraftServer.enablePlugin(CraftServer.java:378) [patched_1.11.2.jar:git-Paper-1106]
        at org.bukkit.craftbukkit.v1_11_R1.CraftServer.enablePlugins(CraftServer.java:328) [patched_1.11.2.jar:git-Paper-1106]
        at net.minecraft.server.v1_11_R1.MinecraftServer.t(MinecraftServer.java:438) [patched_1.11.2.jar:git-Paper-1106]
        at net.minecraft.server.v1_11_R1.MinecraftServer.l(MinecraftServer.java:399) [patched_1.11.2.jar:git-Paper-1106]
        at net.minecraft.server.v1_11_R1.MinecraftServer.a(MinecraftServer.java:337) [patched_1.11.2.jar:git-Paper-1106]
        at net.minecraft.server.v1_11_R1.DedicatedServer.init(DedicatedServer.java:276) [patched_1.11.2.jar:git-Paper-1106]
        at net.minecraft.server.v1_11_R1.MinecraftServer.run(MinecraftServer.java:612) [patched_1.11.2.jar:git-Paper-1106]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_291]
Caused by: ch.njol.skript.SkriptAPIException: Registration can only be done during plugin initialization
        at ch.njol.skript.Skript.checkAcceptRegistrations(Skript.java:1223) ~[?:?]
        at ch.njol.skript.Skript.registerEvent(Skript.java:1429) ~[?:?]
        at ch.njol.skript.events.EvtSkript.<clinit>(EvtSkript.java:41) ~[?:?]
        ... 14 more
```

**Comment by TheLimeGlass** [comment]
Was the potion fix even added to the dev branch was it compiled in 2.6.3? Still getting error
```
[11:56:59 ERROR]: #!#! Stack trace:
[11:56:59 ERROR]: #!#! java.lang.IllegalArgumentException: ItemStack must be a splash potion. This item stack was LINGERING_POTION.
[11:56:59 ERROR]: #!#!     at org.apache.commons.lang.Validate.isTrue(Validate.java:136)
[11:56:59 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_13_R2.entity.CraftSplashPotion.setItem(CraftSplashPotion.java:24)
[11:56:59 ERROR]: #!#!     at ch.njol.skript.entity.ThrownPotionData.spawn(ThrownPotionData.java:116)
[11:56:59 ERROR]: #!#!     at ch.njol.skript.entity.ThrownPotionData.spawn(ThrownPotionData.java:45)
[11:56:59 ERROR]: #!#!     at ch.njol.skript.entity.EntityData.spawn(EntityData.java:417)
[11:56:59 ERROR]: #!#!     at ch.njol.skript.sections.EffSecSpawn.walk(EffSecSpawn.java:160)
[11:56:59 ERROR]: #!#!     at ch.njol.skript.lang.EffectSectionEffect.walk(EffectSectionEffect.java:44)
[11:56:59 ERROR]: #!#!     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:89)
[11:56:59 ERROR]: #!#!     at ch.njol.skript.lang.Trigger.execute(Trigger.java:56)
[11:56:59 ERROR]: #!#!     at ch.njol.skript.SkriptEventHandler.check(SkriptEventHandler.java:142)
[11:56:59 ERROR]: #!#!     at ch.njol.skript.SkriptEventHandler.access$000(SkriptEventHandler.java:53)
[11:56:59 ERROR]: #!#!     at ch.njol.skript.SkriptEventHandler$PriorityListener.lambda$new$0(SkriptEventHandler.java:71)
[11:56:59 ERROR]: #!#!     at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80)
[11:56:59 ERROR]: #!#!     at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:66)
[11:56:59 ERROR]: #!#!     at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:520)
[11:56:59 ERROR]: #!#!     at ch.njol.skript.Skript$1.lambda$run$0(Skript.java:631)
[11:56:59 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftTask.run(CraftTask.java:84)
[11:56:59 ERROR]: #!#!     at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:449)
[11:56:59 ERROR]: #!#!     at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1010)
[11:56:59 ERROR]: #!#!     at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439)
[11:56:59 ERROR]: #!#!     at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:940)
[11:56:59 ERROR]: #!#!     at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:837)
[11:56:59 ERROR]: #!#!     at java.lang.Thread.run(Thread.java:748)
```

**Comment by TheLimeGlass** [comment]
1.19 and 1.12 erroring with the bob test. Unsure if this is something we should address.
```
[12:07:46 ERROR]: Could not pass event SkriptTestEvent to Skript v2.6.3
java.lang.AssertionError: diamond sword named BOB (class ch.njol.skript.aliases.ItemType) != diamond sword named BOB (class ch.njol.skript.aliases.ItemType): [89, 103, 103, 0, 0, 1, -128, 8, 105, 116, 101, 109, 116, 121, 112, 101, -122, 3, 97, 108, 108, 15, 0, 5, 116, 121, 112, 101, 115, -128, 9, 65, 114, 114, 97, 121, 76, 105, 115, 116, -127, 6, 118, 97, 108, 117, 101, 115, 48, -128, 6, 79, 98, 106, 101, 99, 116, -128, 1, -128, 11, 78, 101, 119, 73, 116, 101, 109, 68, 97, 116, 97, -121, 7, 105, 115, 65, 108, 105, 97, 115, 15, 0, 5, 112, 108, 97, 105, 110, 15, 0, 4, 109, 101, 116, 97, -128, 34, 67, 111, 110, 102, 105, 103, 117, 114, 97, 116, 105, 111, 110, 83, 101, 114, 105, 97, 108, 105, 122, 97, 98, 108, 101, 95, 73, 116, 101, 109, 77, 101, 116, 97, -127, 5, 118, 97, 108, 117, 101, 32, -128, 94, 118, 97, 108, 117, 101, 58, 10, 32, 32, 61, 61, 58, 32, 73, 116, 101, 109, 77, 101, 116, 97, 10, 32, 32, 109, 101, 116, 97, 45, 116, 121, 112, 101, 58, 32, 85, 78, 83, 80, 69, 67, 73, 70, 73, 67, 10, 32, 32, 100, 105, 115, 112, 108, 97, 121, 45, 110, 97, 109, 101, 58, 32, 66, 79, 66, 10, 32, 32, 108, 111, 114, 101, 58, 10, 32, 32, 45, 32, 76, 79, 82, 69, 49, 10, 32, 32, 45, 32, 76, 79, 82, 69, 50, 10, 9, 105, 116, 101, 109, 70, 108, 97, 103, 115, 3, 0, 0, 0, 2, 10, 105, 115, 65, 110, 121, 116, 104, 105, 110, 103, 15, 0, 2, 105, 100, 3, 0, 0, 1, 18, 8, 105, 116, 101, 109, 70, 111, 114, 109, 15, 0, 6, 97, 109, 111, 117, 110, 116, 3, 0, 0, 0, 1, 4, 105, 116, 101, 109, 0, 10, 103, 108, 111, 98, 97, 108, 77, 101, 116, 97, 0, 5, 98, 108, 111, 99, 107, 0]
        at ch.njol.skript.registrations.Classes.serialize(Classes.java:735) ~[?:?]
        at ch.njol.skript.variables.Variables.serialize(Variables.java:548) ~[?:?]
        at ch.njol.skript.variables.Variables.serialize(Variables.java:542) ~[?:?]
        at ch.njol.skript.variables.Variables.saveVariableChange(Variables.java:552) ~[?:?]
        at ch.njol.skript.variables.Variables.setVariable(Variables.java:385) ~[?:?]
        at ch.njol.skript.variables.Variables.setVariable(Variables.java:376) ~[?:?]
        at ch.njol.skript.lang.Variable.set(Variable.java:467) ~[?:?]
        at ch.njol.skript.lang.Variable.change(Variable.java:523) ~[?:?]
        at ch.njol.skript.effects.EffChange.execute(EffChange.java:282) ~[?:?]
        at ch.njol.skript.lang.Effect.run(Effect.java:50) ~[?:?]
        at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:61) ~[?:?]
        at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:89) ~[?:?]
        at ch.njol.skript.lang.Trigger.execute(Trigger.java:56) ~[?:?]
        at ch.njol.skript.SkriptEventHandler.check(SkriptEventHandler.java:142) ~[?:?]
        at ch.njol.skript.SkriptEventHandler.access$000(SkriptEventHandler.java:53) ~[?:?]
        at ch.njol.skript.SkriptEventHandler$PriorityListener.lambda$new$0(SkriptEventHandler.java:71) ~[?:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:78) ~[patched_1.12.2.jar:git-Paper-1620]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[patched_1.12.2.jar:git-Paper-1620]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:513) ~[patched_1.12.2.jar:git-Paper-1620]
        at ch.njol.skript.Skript$1.lambda$run$0(Skript.java:631) ~[?:?]
        at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:64) ~[patched_1.12.2.jar:git-Paper-1620]
        at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:423) ~[patched_1.12.2.jar:git-Paper-1620]
        at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:840) ~[patched_1.12.2.jar:git-Paper-1620]
        at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:423) ~[patched_1.12.2.jar:git-Paper-1620]
        at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:774) ~[patched_1.12.2.jar:git-Paper-1620]
        at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:666) ~[patched_1.12.2.jar:git-Paper-1620]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_291]
```

**Comment by TheLimeGlass** [comment]
Lower priority from high to medium because it's legacy versions, not many use them.

**Comment by TPGamesNL** [comment]
The lingering potion error isn't on the dev/2.6 branch, it was merged after the release: https://github.com/SkriptLang/Skript/pull/4904
The 1.9-1.12 tests with BOB fails have been doing that for a long time (https://github.com/SkriptLang/Skript/issues/3767#issuecomment-788198730)
`soundcategory` type is always registered, the exception is likely caused by another exception during enabling

**Comment by TheLimeGlass** [comment]
> The lingering potion error isn't on the dev/2.6 branch, it was merged after the release: #4904 The 1.9-1.12 tests with BOB fails have been doing that for a long time ([#3767 (comment)](https://github.com/SkriptLang/Skript/issues/3767#issuecomment-788198730)) `soundcategory` type is always registered, the exception is likely caused by another exception during enabling

Ok well the lingering potion should be in 2.6.4, and I fixed the sound category, it's actually not being registered because the class doesn't exist, but the stop sound effect is registering the type. I fixed it in the pull mentioned.
